style/style.css

```diff
@@ -0,0 +1,89 @@
+.wrapper{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 10px;
+    margin: 20px;
+}
+.header-logo
+{
+    margin-left: 20px;
+}
+.line
+{
+    border-top: 2px solid #3aa58e;
+    border-bottom: 2px solid #3aa58e;
+    padding: 3px;
+}
+.box{
+    width: 33%;
```

Comment:
jesli na rodzicu tego elementu masz display flex, to zamiast podawac tuttaj rozmiar w %, wystarczy ze dasz tutaj "flex: 1" i boxy beda idealnie takiej samej szerokosci